sparkles/core.py

```diff
@@ -884,6 +884,23 @@ def check_guide_count(self):
                 'caution',
                 f'{obs_type} with more than {bright_cnt_lim} stars brighter than 6.1.')
 
+        n_guide = len(self.guides)
+
+        # Different number of guide stars than requested
+        if n_guide != self.n_guide:
+            self.add_message(
+                'caution',
```

Comment:
I'm going back and forth on if this might be worthy of a 'warning'.

Reply:
Could go either way.  I don't think it matters all that much in the end, and perversely a caution will be more visible than warning because they are unusual.  I just want to be done with this and move on to the next thing.  If you want to make a commit to change then that's fine.